build-aux/m4/bitcoin_qt.m4

```diff
@@ -399,17 +399,17 @@ AC_DEFUN([_BITCOIN_QT_FIND_LIBS_WITH_PKGCONFIG],[
     qt4_modules="QtCore QtGui QtNetwork"
     BITCOIN_QT_CHECK([
       if test x$bitcoin_qt_want_version = xqt5 || ( test x$bitcoin_qt_want_version = xauto && test x$auto_priority_version = xqt5 ); then
-        PKG_CHECK_MODULES([QT], [$qt5_modules], [QT_INCLUDES="$QT_CFLAGS"; have_qt=yes],[have_qt=no])
```

Comment:
Does the first argument here influence anything else besides how it is printed in the logging?
https://developer.gnome.org/anjuta-build-tutorial/stable/library-autotools.html.en
Seems to imply `LIB_CFLAGS` and `LIB_LDFLAGS` get defined based on this parameter.